torch_xla/runtime.py

```diff
@@ -220,8 +220,17 @@ def addressable_runtime_device_count() -> int:
   return torch_xla._XLAC._xla_num_runtime_devices()
 
 
+def disable_spmd():
+  """API to dsiable SPMD mode.
+  Revert the environment variable set by `use_spmd`, reset  
+  """
+  os.environ["XLA_USE_SPMD"] = "0"
+  torch_xla._XLAC._set_spmd_mode(False)
+
+
 # TODO(yeounoh) introduce SPMD configuration.
-def use_spmd(auto: Optional[bool] = False):
+def use_spmd(auto: Optional[bool] = False,
+             force_tensors_on_spmd_device: Optional[bool] = False):
   """API to enable SPMD mode. This is a recommended way to enable SPMD.
 
   This forces SPMD mode if some tensors are already initialized on non-SPMD
```

Comment:
This comment is probably out of date. Also I think whoever wrote this comment had in mind an idea of forcefully enabling SPMD mode, like that's an aggressive act. But with your PR there's nothing forceful about this anymore. IIUC people now have a clear option of replicating the existing tensors, or not replicating them (and keeping them on non-SPMD devices).